src/Types/Transaction.purs

```diff
@@ -70,6 +73,14 @@ instance monoidTransaction :: Monoid Transaction where
     , auxiliary_data: Nothing
     }
 
+-- Do we actually need this?
```

Comment:
I really don't understand this instance or the one below. I can't imagine how it could be used sensibly

Reply:
It's used in `wrapValidator` constraints, although I'm not convinced we need it ATM. I could comment this function out for now?